### PR TITLE
Rename imsOrgId to orgId.

### PR DESCRIPTION
**Pull request by jfkhoury**
## Description

`ims` is an internal concept. Please don't merge until the change is made on Konductor.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ x ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ x ] I have submitted a [documentation](https://github.com/AdobeDocs/alloy-docs) pull request or no changes are needed.
- [ x ] I have made any necessary test changes and all tests pass.
- [ x ] I have run the Sandbox successfully.

### CODE REVIEW

**Comment by artur-ciocanu** [comment]
@jfkhoury why not `organizationId` instead of `orgId` here: https://github.com/adobe/target-nodejs-sdk we already use that.

**Comment by jfkhoury** [comment]
> @jfkhoury why not `organizationId` instead of `orgId` here: https://github.com/adobe/target-nodejs-sdk we already use that.

@artur-ciocanu Justin asked for `orgId`.

File: src/core/createEventManager.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default ({ createEvent, optIn, lifecycle, network, config, logger }) => {
   const addMetaTo = payload => {
     const meta = {
       gateway: {
-        imsOrgId
+        orgId
```
Comment (Aaronius) [inline src/core/createEventManager.js]: I think this is bad, because Konductor is still expecting `imsOrgId`.

Comment (jfkhoury) [reply src/core/createEventManager.js]: @Aaronius We created a ticket for them to rename it and reached out to Rares. We will wait until Konductor has published before we merge this PR.

Comment (Aaronius) [reply src/core/createEventManager.js]: Okay, cool. Thanks.

Comment (Aaronius) [reply src/core/createEventManager.js]: I just saw "Please don't merge until the change is made on Konductor." in your description. I'm sorry for not reading the description. 😞

**Comment by jfkhoury** [comment]
Merging since the change has been made in Konductor.

**Pull request merged**